css/style.css

```
@@ -0,0 +1,903 @@
+  *,
+*::after,
+*::before {
+    box-sizing: border-box;
+}
+
+html {
+    scroll-behavior: smooth;
+    /* свойство scroll-behavior не наследуется, применяется к прокручиваемым блокам */
+}
+
+ul {
+    list-style: none;
+}
+
+a {
+    text-decoration: none;
+    color: inherit;
+    display: inline-block;
+}
+
+.header {
+    font-family: Montserrat, sans-serif;
+    height: 806px;
+    background-color: #eff0f2;
+}
+
+.container {
+    width: 1140px;
+    margin: 0 auto;
+}
+
+.header_logo {
+    margin-left: 0%;
+    padding-top: 5.7%;
+}
+
+.header_text {
+    margin-top: 16.7%;
+    margin-left: 5%;
+    text-align: center;
+    margin-bottom: 10%;
+}
+
+.header_text__style {
+    margin-top: 200px;
+    margin-bottom: 137px;
+    text-align: center;
+    font-size: 45px;
+    line-height: 59px;
+    font-weight: 700;
+    text-transform: uppercase;
+    letter-spacing: 2px;
+    text-shadow: 4px 8px 10px rgb(0 0 0 / 23%);
+    color: #505050;
+}
+
+.header_text__style span {
+    background-color: #76f4ea;
+    padding: 0 15px;
+}
+
+.header_button {
+    display: flex;
+    justify-content: center;
+    /* padding-top: 12%; */
+    margin-left: 0%;
+}
+
+.header_button__style {

```

Class naming here and beyond should be semantic. Fix it all.

---

Thanks! 
fixed all class naming
```suggestion
.header-text__style {
  .header-text__title {
```

---

Thanks!
fixed all
```suggestion 
.header-button__style:hover {
  .header-button__contact:hover {
```

---

Thanks!
fixed all
```suggestion
.header-button__style {
  .header-button__contact {
```

---

Thanks!
fixed all!
```suggestion
.header-button__contact {
```